polartools/load_data.py

```diff
@@ -203,21 +203,23 @@ def load_table(scan, source, **kwargs):
     :func:`polartools.load_data.load_spec`
     """
 
-    folder = kwargs.pop("folder", "")
+    _kwargs = copy.deepcopy(kwargs)
```

Comment:
After adding this, do you still need the `kwargs_scan` in the `diffraction.py` functions?

Reply:
Should not be needed anymore. Changed it back to **kwargs

Reply:
also implemented your other comments